README.md

```diff
@@ -18,9 +18,8 @@ Exadel Smart Library (**ESL**) is a web components based library that gives you
 - ##### [ESL Base Trigger](src/modules/esl-base-trigger/README.md)
 - ##### [ESL Popup](src/modules/esl-popup/README.md)
 - ##### [ESL Trigger](src/modules/esl-trigger/README.md)
```

Comment:
Cosmetic (separate): Add a11y group component to the global doc, please

Reply:
Done, thanks